.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ workflows:
           prj_dir: aineko
           python_version: << pipeline.parameters.python_version >>
           markers: integration
+      - test-aineko-create
```

Comment:
we should probably put test-aineko-create and integration-test after python-checks so we only run it if those succeed.

Reply:
Good idea, I'll raise a PR.